### DISCUSSION TITLE
Users Notifications

### DISCUSSION THREAD

**Discussion by falsakran** [neos-sdi/adfsmfa#209] [Q&A] [answered]
Hello @redhook62 

It's great solution for 2FA and easy to use but in my case i need any user enroll he's self receive email to let hem know that your user has been enrolled  with out receiving code only notifications , how can i do it ?

Thank you so much for your great work.

**Comment by redhook62** [comment, marked as answer]
Hi @falsakran 

If you want to register your users yourself (pre-provisioning) and tell them that they are registered and send them the associated QRCode.

You can use import functions from ADDS.

in PowerShell as indicated here: [Import from ADDS](https://github.com/neos-sdi/adfsmfa/wiki/09-Users%20Management#importing-users-from-adds) 
you will notice that it is necessary to indicate the parameters **-NewKey** and **-SendMail** and especially that your user has an email address in his ADDS properties. The destination email provided must be a valid email and not subject to MFA.
If this is not possible from ADDS, you can look at the CSV or Xml imports side, sample files are provided in **C:\Program Files\MFA\Imports**

You can automate this with the Windows task manager (Task Scheduler) as in examples 3 and 4.
Only run this script on one ADFS server at a time.

Tell me if it suits you ?

regards

redhook
